cluster-api/src/main/java/io/aiven/klaw/clusterapi/utils/ClusterApiUtils.java

```diff
@@ -0,0 +1,426 @@
+            env.getProperty(
+                clusterIdentification.toLowerCase()
+                    + ".kafkasasl.saslmechanism.gssapi.servicename"));
+      }
+    } catch (Exception exception) {
+      log.error("Error : Cannot set SASL SSL GSSAPI Config properties.");
+    }
+
+    return props;
+  }
+
+  public Properties getSslConfig(String clusterIdentification) {
+    Properties props = new Properties();
+
+    try {
+      if (!Strings.isNullOrEmpty(
+          env.getProperty(clusterIdentification.toLowerCase() + ".kafkassl.keystore.location"))) {
+        props.put(
+            SslConfigs.SSL_KEYSTORE_LOCATION_CONFIG,
+            env.getProperty(clusterIdentification.toLowerCase() + ".kafkassl.keystore.location"));
+      }
+      if (!Strings.isNullOrEmpty(
+          env.getProperty(clusterIdentification.toLowerCase() + ".kafkassl.keystore.pwd"))) {
+        props.put(
+            SslConfigs.SSL_KEYSTORE_PASSWORD_CONFIG,
+            env.getProperty(clusterIdentification.toLowerCase() + ".kafkassl.keystore.pwd"));
+      }
+      if (!Strings.isNullOrEmpty(
+          env.getProperty(clusterIdentification.toLowerCase() + ".kafkassl.key.pwd"))) {
+        props.put(
+            SslConfigs.SSL_KEY_PASSWORD_CONFIG,
+            env.getProperty(clusterIdentification.toLowerCase() + ".kafkassl.key.pwd"));
+      }
+      if (!Strings.isNullOrEmpty(
+          env.getProperty(clusterIdentification.toLowerCase() + ".kafkassl.keystore.type"))) {
+        props.put(
+            SslConfigs.SSL_KEYSTORE_TYPE_CONFIG,
+            env.getProperty(clusterIdentification.toLowerCase() + ".kafkassl.keystore.type"));
+      } else {
+        props.put(SslConfigs.SSL_KEYSTORE_TYPE_CONFIG, "JKS");
+      }
+      if (!Strings.isNullOrEmpty(
+          env.getProperty(clusterIdentification.toLowerCase() + ".kafkassl.truststore.type"))) {
+        props.put(
+            SslConfigs.SSL_TRUSTSTORE_TYPE_CONFIG,
+            env.getProperty(clusterIdentification.toLowerCase() + ".kafkassl.truststore.type"));
+      } else {
+        props.put(SslConfigs.SSL_TRUSTSTORE_TYPE_CONFIG, "JKS");
+      }
+
+      if (!Strings.isNullOrEmpty(
+          env.getProperty(clusterIdentification.toLowerCase() + ".kafkassl.truststore.location"))) {
+        props.put(
+            SslConfigs.SSL_TRUSTSTORE_LOCATION_CONFIG,
+            env.getProperty(clusterIdentification.toLowerCase() + ".kafkassl.truststore.location"));
+      }
+
+      if (!Strings.isNullOrEmpty(
+          env.getProperty(clusterIdentification.toLowerCase() + ".kafkassl.truststore.pwd"))) {
+        props.put(
+            SslConfigs.SSL_TRUSTSTORE_PASSWORD_CONFIG,
+            env.getProperty(clusterIdentification.toLowerCase() + ".kafkassl.truststore.pwd"));
+      }
+
+      props.put("ssl.enabled.protocols", "TLSv1.2,TLSv1.1");
+      props.put("ssl.endpoint.identification.algorithm", "");
+    } catch (Exception exception) {
+      log.error("Error : Cannot set SSL Config properties.");
```

Comment:
```suggestion
      log.error("Error : Cannot set SSL Config properties.", exception);
```
exception itself in logs would be helpful